infra/routes/label_evaluator_viz_cache/lambdas/cache_generator.py

```
@@ -0,0 +1,392 @@
+        )
+    return {}
+
+
+def _build_receipt_visualization(
+    trace_data: dict[str, Any],
+) -> dict[str, Any] | None:
+    """Build visualization-ready receipt data from trace + DynamoDB data."""
+    image_id = trace_data.get("image_id")
+    receipt_id = trace_data.get("receipt_id")
+    merchant_name = trace_data.get("merchant_name", "Unknown")
+    execution_id = trace_data.get("execution_id", "")
```

Ruff (0.14.10)

Local variable `execution_id` is assigned to but never used

Remove assignment to unused variable `execution_id`

(F841)